packages/sheets-numfmt/src/base/const/CURRENCY-SYMBOLS.ts

```diff
@@ -43,3 +45,21 @@ export const currencySymbols = [
     '₾',
     '₿',
 ];
+
+export const currencyMap: Record<string, string> = {
+    [LocaleType.EN_US]: '$', // United States Dollar
+    CA: 'C$', // Canadian Dollar
+    GB: '£', // British Pound Sterling
+    JP: '¥', // Japanese Yen
+    IN: '₹', // Indian Rupee
+    AU: 'A$', // Australian Dollar
+    [LocaleType.ZH_CN]: '¥', // Chinese Yuan
+    [LocaleType.ZH_TW]: '¥', // Chinese Yuan
+    KR: '₩', // South Korean Won
+    [LocaleType.RU_RU]: '₽', // Russian Ruble
+    // Euro countries
+    AT: '€', BE: '€', CY: '€', EE: '€', FI: '€', FR: '€',
+    DE: '€', GR: '€', IE: '€', IT: '€', LV: '€', LT: '€',
+    LU: '€', MT: '€', NL: '€', PT: '€', SK: '€', SI: '€', ES: '€',
```

Comment:
Is it normal that two-letters country codes are mixed with full locale strings like `en-US` ?

Reply:
Here, currency types are represented using language codes instead of coutry codes (You can find specific language codes in the file located at (packages/core/src/types/enum/locale-type.ts)[https://github.com/dream-num/univer/blob/dev/packages/core/src/types/enum/locale-type.ts]).

Reply:
Yes, and that's a problem, because the currency depends on the country, but not on the language